tools/cockpit.spec

```diff
@@ -262,10 +262,12 @@ rm -rf %{buildroot}/debug
 
 %files pcp
 %{_libexecdir}/cockpit-pcp
+/var/lib/pcp/config/pmlogconf/tools/cockpit
 
 %post pcp
 # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1185749
 ( cd /var/lib/pcp/pmns && ./Rebuild -du )
+/usr/share/pcp/lib/pmlogger reload
```

Comment:
I think it does start it, but only when it is enabled.

PCP seems to have the rule that as long as pmlogger is enabled (in the sense of systemctl is-enabled), pmlogger is supposed to be running.  A cron job will start a pmlogger process, for example.  So starting a pmlogger process when pmlogger is enabled but wasn't running is expected.

To be ready for a future with non-broken pcp integration with systemd, we could change this to "systemctl reload-or-try-restart pmlogger", but that might currently run into https://bugzilla.redhat.com/show_bug.cgi?id=1185764.

So, all things considered, I would keep `/usr/share/pcp/lib/pmlogger reload` in the post script.

I'll add a HACK comment why we can't use `systemctl reload-or-try-restart`.